src/custom-elements/src/utils/utils.ts

```diff
@@ -22,3 +22,10 @@ export function calculateClasses(opts?: CalculateClassesParam): string {
     return '';
   }
 }
+
+let idCounter = 0;
+const idDomain = Math.random().toString(36).replace(/[^a-z]+/g, '').substr(2, 10);
+
+export function getNewUniqueId(): string {
+  return `chi-${idDomain}-${idCounter++}`;
+}
```

Comment:
where are these getting utilized.

Reply:
This was used in an earlier stage of development but finally not included because it wasn't necessary. However, I think it can be useful for future components, that's why I decided to leave it there. Thoughts @dani-cl-madrid, @srinath-koneru?